src/frontend/js/data/useCourseEnrollment/index.ts

```diff
@@ -20,34 +21,42 @@ const useCourseEnrollment = (resourceLink: string) => {
   const queryClient = useQueryClient();
   const EnrollmentAPI = EnrollmentApiInterface(resourceLink);
 
-  const [{ data: enrollment, isError }, queryKey] = useSessionQuery(
+  const [{ data: enrollment, isError, isLoading }, enrollmentKeys] = useSessionQuery(
```

Comment:
Hmmm... I find that `queryKey` is a better name as this is something related to react-query nope ?